common/types/shared.go

```diff
@@ -3646,64 +3646,16 @@ func (v *History) GetEvents() (o []*HistoryEvent) {
 
 // HistoryBranch is an internal type (TBD...)
 type HistoryBranch struct {
-	TreeID    *string               `json:"treeID,omitempty"`
-	BranchID  *string               `json:"branchID,omitempty"`
-	Ancestors []*HistoryBranchRange `json:"ancestors,omitempty"`
```

Comment:
Is it ok to drop the JSON tags? I dont' know if this data is persisted, but I'd be worried that if it is, that'd be a problem

Reply:
It is safe. As mentioned in the description of PR, these types are not serialized directly.
We do serialize thrift counterparts in several places, but not those types.

Reply:
Sorry, 🤦  didn't read